HW1/task2.py

```diff
@@ -0,0 +1,10 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+
+n = int(input("Введите трехзначное число:"))
+n = int(n)
+
+a1 = n % 10
+n = n // 10
+a2 = n % 10
+n = n // 10
```

Comment:
Можно записать на одну строку меньше.
```suggestion
a1 = n % 10
a2 = n // 10 % 10
n = n // 100
```